docs/console.html

```diff
@@ -0,0 +1,804 @@
+{{+bindTo:partials.standard_devtools_article}}
+
+<h1 id="using-the-console">Using the Console</h1>
+
+<p>
+    Utilizing the console gives you the ability to:
+</p>
+<ul>
+    <li>Log diagnostic information to help debug your web page or application.</li>
+    <li>Enter commands that interact with the document and the Chrome DevTools.</li>
+</ul>
+<p>
+    You also may evaluate normal JavaScript expressions.
+    This documentation provides an overview and common uses of the console.
+    You may browse the <a href="console-api">Console API</a> and <a href="commandline-api">Command Line API</a> reference material to understand more functionality than is described here.
+</p>
+
+
+<div class="collapsible">
+    <h2 id="basic-operation">Basic Operation</h2>
+
+
+    <h3 id="opening-the-console">Opening the Console</h3>
+    <p>
+        The JavaScript Console is available in two places.
+        Primary access is provided in the Console panel.
+        It also is available from within any other panel by using the Drawer.
+        To open the Console tab, do one of the following:
+    </p>
+    <ul>
+        <li>Use the keyboard shortcut <kbd><kbd class="kbd">Command</kbd> + <kbd class="kbd">Option</kbd> + <kbd class="kbd">J</kbd></kbd> (Mac) or <kbd><kbd class="kbd">Control</kbd> + <kbd class="kbd">Shift</kbd> + <kbd class="kbd">J</kbd></kbd> (Windows/Linux).</li>
+        <li>Select <samp><img src="devtools/images/chrome-menu.png" alt="Chrome Menu"> > More Tools > JavaScript Console</samp>.</li>
+    </ul>
+    
+    <figure>
+        <img alt="Console panel view" src="console-files/console1.png">
+        <figcaption>
+            A clean console view.
+        </figcaption>
+    </figure>
+        
+    <p>
+        To open the drawer console press the <kbd class="kbd">Esc</kbd> key on your keyboard or click the <samp>Show Drawer</samp> button in the upper right corner of the Chrome DevTools window.
+    </p>
+    
+    <figure>
+        <img alt="Console split-view" src="console-files/console-in-drawer.png">
```

Comment:
this is a really messy looking screenshot. can we do something more simple? bottom-dock/sources/drawer.  no existing exceptions.